### PR TITLE
fix(ui): pipeline styles adapted to a tabler

### LINKED ISSUE #72: Fix Pipelines styles
**Problem**
Current Pipeline columns looks bad:

CleanShot 05 11 2024 at 13-06

**Solution**
Mockups: [Figma](https://www.figma.com/design/kQOH4s5DnfWL0p6o6tlMQz/Draft-%5BATS%5D?node-id=1442-6528&t=8OO6JZm6OtBkLOzn-1)

For Pipeline columns:
- Set #DCE1E7 border-color
- Set #FFFFFF background color
- Set default badge styles for each counter
- Try to apply Tabler's Buttons group style for Tab buttons (qualified/reserved/disqualified)

CleanShot 05 11 2024 at 13-05

Related: https://github.com/freeats/freeats/issues/16

### PR DESCRIPTION
**Pull request by ksnitsky**
Closes #72.

- [x] I have reviewed my code in "Files changed" tab.
- [x] I have re-read the issue and this PR fully resolves it.

### CODE REVIEW

**Comment by ksnitsky** [comment]
I didn't have mockups of the mobile version, so I made it as it was, but with the new styles

---

The bottom borders are shadows for which there is already an issue https://github.com/freeats/freeats/issues/51 to remove them
image

File: app/assets/stylesheets/src/ats/_positions.scss [hunk 1]
```diff
@@ -26,84 +26,90 @@
...
-      .avatar {
-        --tblr-avatar-icon-size: 2.5rem;
-      }
-      img {
-        width: 100%;
-        height: auto;
-      }
+    @include media-breakpoint-up(sm) {
+      background-color: $card-bg;
+      border: $card-border-width solid $card-border-color;
```
Comment (ksnitsky) [inline app/assets/stylesheets/src/ats/_positions.scss:32]: https://github.com/freeats/freeats/issues/72#issuecomment-2458805335

The standard color of the card borders `rgba(4, 32, 69, 0.1)` is used here

**Review by wrmk — APPROVED** [review]
looks different
image

mockup
image

**Comment by wrmk** [comment]
> 
> On the mockup still with fa user icon and in this PR I didn't touch the cards. Avatars in them were fixed in #42. I assume we are happy with the standard avatar styles from tabler.

let's merge then

**Pull request merged**